.github/workflows/build.yml

```diff
@@ -112,8 +114,8 @@ jobs:
       - name: cargo test (debug; rustls-provider-test; all features)
         run: cargo test --all-features -p rustls-provider-test
 
-      - name: cargo package --all-features -p ${{ env.MAIN_CRATE }}
-        run: cargo package --all-features -p ${{ env.MAIN_CRATE }}
+      # FOR FUTURE CONSIDERATION: ADD NICE name for this step
+      - run: cargo package ${{ env.ALL_FEATURES_EXCEPT_UNSAFE }} -p ${{ env.MAIN_CRATE }}
```

Comment:
I would love to get a better understanding of the effect of doing `cargo package` with feature options such as no features, `--all-features`, `--features unstable-api-all`, etc. etc. I think there should be a TODO comment for this.